src/main/kotlin/no/nav/familie/ef/iverksett/arena/VedtakhendelseMapper.kt

```diff
@@ -0,0 +1,35 @@
+package no.nav.familie.ef.iverksett.arena
+
+import no.nav.familie.ef.iverksett.iverksetting.domene.Iverksett
+import no.nav.melding.virksomhet.vedtakhendelser.v1.vedtakhendelser.WSApplikasjoner
+import no.nav.melding.virksomhet.vedtakhendelser.v1.vedtakhendelser.WSAvslutningsstatuser
+import no.nav.melding.virksomhet.vedtakhendelser.v1.vedtakhendelser.WSBehandlingstema
+import no.nav.melding.virksomhet.vedtakhendelser.v1.vedtakhendelser.WSVedtakHendelser
+import java.text.SimpleDateFormat
+import java.util.*
+import javax.xml.datatype.DatatypeFactory
+
+
+fun mapIverksettTilVedtakHendelser(iverksett: Iverksett): WSVedtakHendelser {
+    val wsVedtakHendelser = WSVedtakHendelser()
+
+    val wsAvslutningsstatuser = WSAvslutningsstatuser()
+    wsAvslutningsstatuser.kodeRef = iverksett.vedtak.vedtaksresultat.name
+    wsVedtakHendelser.avslutningsstatus = wsAvslutningsstatuser //Arena støtter ikke avslått?
+
+    val wsBehandlingstema = WSBehandlingstema()
+    wsBehandlingstema.value = "ab0272" //Enslig forsørger
```

Comment:
Litt usikker her, virker som att det finnes lite ulike behandlingstemaer her? 
https://confluence.adeo.no/display/FEL/Sak+og+Behandling+-+Grensesnitt+til+Arena har behandlingstema både for "Enslig forsørger" og "Overgangsstønad"

Vi har noen her som går å bruke
https://github.com/navikt/familie-kontrakter/blob/master/felles/src/main/kotlin/no/nav/familie/kontrakter/felles/Behandlingstema.kt#L13

Hvis det er ab0272 kanskje du har mulighet å legge inn den i kontrakter å bruke den enumen? :)

Reply:
Nå husker jeg ikke eksakt hva Arena trengte, er det de som trenger at man har "rett til overgangsstønad" (?) Isåfall er det alltid ab0272 som skal brukes, men det må kanskje sjekkes ut.

Reply:
Ja det gir vel mening, men sikkert fint å sjekke!
Tror vi sansynligvis skal sende den samme når vi har fattet vedtak på noen av de andre 2 stønadstypene også, men då kanskje bruke den samme koden

Reply:
Avklarte med Arena. Vi skal bruke behandlingstema (Enslig forsørger / ab0272 var visst ikke i bruk lenger) - har tatt hensyn til dette i mappingen. Rettet dette til å nå bruke Behandlingstema fra familie felles kontrakter i mappingen.